app/src/display/widgets/Kconfig

```diff
@@ -32,12 +32,16 @@ config ZMK_WIDGET_PERIPHERAL_STATUS
     bool "Widget for split peripheral status icons"
     depends on BT && ZMK_SPLIT_BLE && !ZMK_SPLIT_ROLE_CENTRAL
     default y if BT && ZMK_SPLIT_BLE && !ZMK_SPLIT_ROLE_CENTRAL
-    select LV_USE_LABEL
```

Comment:
Is this an intentional change?

Reply:
Good catch! No, this was completely unintentional and I cannot explain this!